### ISSUE TITLE
[0.1.8] TypeError: super() takes at least 1 argument (0 given)

### ISSUE THREAD

**Issue by DeepDiver1975** [owncloud/ocdev#44]
see https://travis-ci.org/owncloud/calendar-rework/jobs/98094074
````
Traceback (most recent call last):

  File "/home/travis/.local/bin/ocdev", line 9, in <module>

    load_entry_point('ocdev==0.1.8', 'console_scripts', 'ocdev')()

  File "build/bdist.linux-x86_64/egg/pkg_resources/__init__.py", line 519, in load_entry_point

  File "build/bdist.linux-x86_64/egg/pkg_resources/__init__.py", line 2630, in load_entry_point

  File "build/bdist.linux-x86_64/egg/pkg_resources/__init__.py", line 2310, in load

  File "build/bdist.linux-x86_64/egg/pkg_resources/__init__.py", line 2316, in resolve

  File "/home/travis/.local/lib/python2.7/site-packages/ocdev/application.py", line 20, in <module>

    from ocdev.plugins import PLUGINS

  File "/home/travis/.local/lib/python2.7/site-packages/ocdev/plugins/__init__.py", line 8, in <module>

    PLUGINS = [SetUp(), StartApp(), ContinousIntegration(), Server(), DevUp()] # AppStore()

  File "/home/travis/.local/lib/python2.7/site-packages/ocdev/plugins/setup/setup.py", line 22, in __init__

    super().__init__('setup')

TypeError: super() takes at least 1 argument (0 given)

The command "ocdev setup core --dir owncloud --branch $CORE_BRANCH --no-history" failed and exited with 1 during .

**Comment by BernhardPosselt** [comment]
You are using Python 2, ocdev requires Python 3, check the paths in your log ;)

**Comment by DeepDiver1975** [comment]
Understood that meanwhile as well - THX

Would be great to have an explicit message stating this instead of a stack trace ...

**Comment by BernhardPosselt** [comment]
Fixed in 0.2.0 which is now live
